### ISSUE TITLE
[4.0.0-beta.16] ion-chip-button is not a known element

### ISSUE THREAD

**Issue by hebra** [ionic-team/ionic#16352]
# Bug Report

**Ionic Info**
Run `ionic info` from a terminal/cmd prompt and paste the output below.

```
Ionic:

   ionic (Ionic CLI)             : 4.3.1 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.0.0-beta.16
   @angular-devkit/build-angular : 0.10.6
   @angular-devkit/schematics    : 7.0.6
   @angular/cli                  : 7.0.6
   @ionic/angular-toolkit        : 1.2.0

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : android 7.1.1, browser 5.0.4, ios 4.5.5
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.2, cordova-plugin-ionic-webview 2.0.3, (and 18 other plugins)

System:

   Android SDK Tools : 26.1.1 (/Users/heb/Library/Android/sdk)
   ios-deploy        : 1.9.4
   NodeJS            : v11.1.0 (/usr/local/Cellar/node/11.1.0/bin/node)
   npm               : 6.4.1
   OS                : macOS
   Xcode             : Xcode 10.1 Build version 10B61
```

**Describe the Bug**
With the beta.16 update the element ion-chip-button is suddenly unkonwn. ion-chip itself works, and with beta.15 or earlier the ion-chip-button also works.

**Steps to Reproduce**
1. In a template file add an ion-chip with an ion-chip-button inside like this:
```
            <ion-chip>
                <ion-label>Chips</ion-label>
                <ion-chip-button>
                    <ion-icon name="close-circle"></ion-icon>
                </ion-chip-button>
            </ion-chip>
```
2. Load the template in a browser
3. Loading fails to the reported issuse, when downgrading to beta.15 it works

**Expected Behavior**
As nothing related to ion-chip was mentioned in the beta.16 release notes I assume that this should still work as before.

**Comment by devsaran** [comment]
Same issue here too.

**Comment by paulstelzer** [comment]
You are right, it was not mentioned in changelog. They refactored ion-chip, see this commit: #bf96260

**Comment by manucorporat** [comment]
I am sorry for the inconvenience, we will update the CHANGELOG asap, leaving open until the it's updated.

**Comment by redwolf2** [comment]
Use `ion-button` and `ion-icon` instead, will not create a round button, but a rectangular one.

**Comment by camwiegert** [comment]
@redwolf2 We've removed that round chip-button style because it is no longer in the Material Design spec. You can see the [web implementation of the new chip spec here](https://material-components.github.io/material-components-web-catalog/#/component/chips).

I would recommend nesting an icon directly in the chip. You could use the `close-circle` icon for a similarly rounded icon.

```html
<ion-chip>
  <ion-label>With Close Icon</ion-label>
  <ion-icon name="close-circle"></ion-icon>
</ion-chip>
```

<img width="167" alt="screen shot 2018-11-21 at 11 04 19 am" src="https://user-images.githubusercontent.com/2547860/48857001-7e45d200-ed7d-11e8-8974-5fb8a96f5a17.png">

**Comment by redwolf2** [comment]
@camwiegert Ok, thx for the suggestion.

**Comment by simonhaenisch** [comment]
@camwiegert I have issues with the chip when wrapping it inside an `ion-item`:

<img src="https://user-images.githubusercontent.com/3410759/48873858-15ca2400-ee55-11e8-85c1-aa5703dad68e.png" width="190">

It happens because the `ion-label` gets a margin when inside an item:

<img src="https://user-images.githubusercontent.com/3410759/48874019-d05a2680-ee55-11e8-9fd5-4225d277396c.png" width="375">

`<ion-label no-margin>` doesn't fix it, it has to be `<ion-label style={{ margin: '0' }}>`.

---

Thanks for updating the changelog (we really rely on that while developing a production app on a beta version).
